golem-base-indexer/golem-base-indexer-logic/src/mat_view_scheduler.rs

```diff
@@ -0,0 +1,64 @@
+use std::sync::Arc;
```

Comment:
We a already have something like this: https://github.com/Golem-Base/blockscout-rs-neti/blob/main/golem-base-indexer/golem-base-indexer-logic/src/updater_leaderboards.rs#L17

Can we unify it? Seems to have exactly the same point.